cmd/build/helmify/replacements.go

```diff
@@ -221,7 +221,7 @@ var replacements = map[string]string{
 
 	"HELMSUBST_MUTATING_WEBHOOK_MATCH_CONDITIONS": `{{ toYaml .Values.mutatingWebhookMatchConditions | nindent 4 }}`,
 
-	"HELMSUBST_PDB_CONTROLLER_MANAGER_MINAVAILABLE": `{{ .Values.pdb.controllerManager.minAvailable }}`,
+	"HELMSUBST_PDB_CONTROLLER_MANAGER": `{{ .Values.pdb.controllerManager | toYaml | nindent 2 }}`,
```

Comment:
Unfortunately this does not ensure backward compat. Instead please add a new value for maxUnavailable and if/else block to ensure only one of them is set.


e.g. something like: 

{{- if and .Values.maxAvailable .Values.maxUnavailable }}
  {{ fail "Only one of 'maxAvailable' or 'maxUnavailable' can be set, but not both" }}
{{- else if not (or .Values.maxAvailable .Values.maxUnavailable) }}
  {{ fail "One of 'maxAvailable' or 'maxUnavailable' must be set" }}
{{- else }}
  # Continue with resource configuration
  {{ if .Values.maxAvailable }}
    maxAvailable: {{ .Values.maxAvailable }}
  {{ else if .Values.maxUnavailable }}
    maxUnavailable: {{ .Values.maxUnavailable }}
  {{ end }}
{{- end }}

Reply:
okay. done.